config/database.yml

```diff
@@ -29,11 +29,17 @@ test:
 # the names of the vars we're looking for without ensuring you've
 # updated/duplicated the vars there!
 production:
-  adapter: postgresql
-  encoding: unicode
-  host: <%= ENV["DIAPER_DB_HOST"] %>
-  pool: 5
-  username: <%= ENV["DIAPER_DB_USERNAME"] %>
-  password: <%= ENV["DIAPER_DB_PASSWORD"] %>
-  database: <%= ENV["DIAPER_DB_DATABASE"] %>
-  timeout: 5000
+  primary: &primary_production
+    <<: *default
+    adapter: postgresql
+    encoding: unicode
+    host: <%= ENV["DIAPER_DB_HOST"] %>
+    pool: 5
+    username: <%= ENV["DIAPER_DB_USERNAME"] %>
+    password: <%= ENV["DIAPER_DB_PASSWORD"] %>
+    database: <%= ENV["DIAPER_DB_DATABASE"] %>
+    timeout: 5000
+  cache:
```

Comment:
If we're creating a separate database... that needs a script or migration to be run, correct?

Reply:
All I had to run (with my env set to production) was `db:prepare` and that created the solid cache database and created the cache table.

I haven't really looked into the inner workings of how the database is created and table schema loaded.

I assume the deployment process for production has a `db:migrate` step but i dont think that will be enough in this case. Will double check tomorrow

Reply:
Yea so `db:prepare` in this case creates the database and then loads the cache table from the new schema file `db/cache_schema.rb`. It would also run any pending migrations, but there aren't any for this new cache table, just the schema to load.

Reply:
@coalest can you look to see if we can use the same database instead of 2 different ones? I know the docs seem to favor a separate DB, but it would simplify migration if we could just create a table in the existing DB.

Reply:
Yea I believe using the same database should be fine. I will make that change.

Reply:
Fixed in 8b116b61551ba27a0d42761b6974a9d55ac36545.